homeassistant/components/pyload/api.py

```diff
@@ -0,0 +1,107 @@
+"""Simple wrapper for pyLoad's API."""
+
+from http import HTTPStatus
+from json import JSONDecodeError
+import logging
+import traceback
+from typing import Optional
+
+import aiohttp
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class PyLoadAPI:
```

Comment:
Please extract the device specific interface code and data parsing to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices